src/centroidal_pd_controller.cpp

```diff
@@ -82,8 +82,14 @@ void CentroidalPDController::run(Eigen::Ref<const Eigen::Vector3d> kc,
 
     /*---------- computing ang error ----*/
 
+    // Rotate the des and current angular velocity into the world frame.
+    angvel_world_error_ = ori_se3_ * angvel;
+    des_angvel_world_error_ = des_ori_se3_ * angvel_des;
```

Comment:
We expect the angular velocity to be expressed in the local frame. Therefore, doing the conversion from local to global frame here.